doc/sfp-cmis/Interface-Link-bring-up-sequence.md

```diff
@@ -184,18 +185,246 @@ if transceiver is not present:
  - All the workflows mentioned above will reamin same ( or get exercised) till host_tx_ready field update
  - xcvrd will not perform any action on receiving host_tx_ready field update 
 
-
+# Port reinitialization during syncd/swss/orchagent crash
+## Overview
+
+When syncd/swss/orchagent crashes, all ports in the corresponding namespace will be reinitialized by xcvrd irrespective of the current state of the port.  
+If just xcvrd crashes and restarts, then forced reinitialization (CMIS reinit + media settings notify) of port will not be performed.  
+Following infra will ensure port reinitialization by xcvrd in case of syncd/swss/orchagent crash:
+
+1. XCVRD main thread init
+	- XCVRD main thread creates the key CMIS_REINIT_REQUIRED in PORT_TABLE:\<port\> (APPL_DB) with value as true for ports which do NOT have this key present 
```

Comment:
Isn't this the PORT_TABLE of STATE_DB?

Reply:
I think the confusion here is because PORT_TABLE exists in APPL_DB as well as STATE_DB (although, both are unique). I have now modified the below line to be more precise to APPL_DB
"Check for changes in PORT_TABLE and act upon receiving DEL event" -> "Check for changes in APPL_DB:PORT_TABLE and act upon receiving DEL event"